### PR TITLE
Add ADR about TAD standard

### PR DESCRIPTION
**Pull request by ChristopherSpelt**
# Description

Created an ADR directory for TAD with an ADR about our TAD Reporting Standard. 

## Checklist

Please check all the boxes that apply to this pull request using "x":

- [x]  I have tested the changes locally and verified that they work as expected.
- [x]  I have followed the project's coding conventions and style guidelines.
- [x]  I have rebased my branch onto the latest commit of the main branch.
- [x]  I have squashed or reorganized my commits into logical units.
- [x]  I have read, understood and agree to the Developer Certificate of Origin below, which this project utilises.

## Developer Certificate of Origin

Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.

### CODE REVIEW

File: docs/Projects/TAD/ADRs/0002-tad-reporting-standard.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+# TAD-0002 TAD Reporting Standard
+
+## Context
+
+The TAD Reporting Standard proposes a standardised way of capturing information of ML-models and systems.
+
+## Assumptions
+
+There does not exists a standard of capturing all relevant information on ML-models that also includes
```
Comment (uittenbroekrobbert) [inline docs/Projects/TAD/ADRs/0002-tad-reporting-standard.md:9]:
```suggestion
There is no exists a standard of capturing all relevant information on ML-models that also includes
```

File: docs/Projects/TAD/ADRs/0002-tad-reporting-standard.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+fairness and bias tests and regulatory assessments.
+
+A widely used implementation for [Model Cards for Model Reporting](https://arxiv.org/abs/1810.03993) is given
+by the [Hugging Face Model Card metadata specification](https://github.com/huggingface/hub-docs/blob/main/modelcard.md?plain=1),
+which in turn is based on [Papers with Code Model Index](https://github.com/paperswithcode/model-index).
+This implementation does not include fields for bias and fairness tests or regulartory assessments.
```
Comment (uittenbroekrobbert) [inline docs/Projects/TAD/ADRs/0002-tad-reporting-standard.md:15]:
```suggestion
This implementation does not include fields for measurements and metrics from technical tests like bias and fairness tests or regulatory assessments.
```

File: docs/Projects/TAD/ADRs/0002-tad-reporting-standard.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+## Decision
+
+We decided to implement a custom reporting standard. Our reporting standard can be split up
+into three elements.
+
+1. System Card, containing information about a group of ML-models which accomplish a specific task.
+A System Card can refer to multiple Model Cards, either a Model Card specified by the TAD Reporting
+Standard, or any other model card. A System Card can refer to multiple Assessment Cards.
+2. Model Card, containing information about a specific ML-model.
```
Comment (uittenbroekrobbert) [inline docs/Projects/TAD/ADRs/0002-tad-reporting-standard.md:25]: I wonder if we also make measurements and metrics separate includes, so results in their own file, as we also do with assessments.

File: docs/Projects/TAD/ADRs/0002-tad-reporting-standard.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+
+We were heavily inspired by the Hugging Face Model Card metadata specification, which we essentially
+extended to allow for:
+
+1. More finegrained information on performance metrics.
+2. Capturing additional measurements on fairness and bias.
+3. Capturing regulartory assessments.
+
+The extension is not strict, meaning that there the TAD Reporting Standard is not a valid Hugging Face
+metadata specification. The reason for this is that some fields in the Hugging Face standard to too much
```
Comment (uittenbroekrobbert) [inline docs/Projects/TAD/ADRs/0002-tad-reporting-standard.md:36]:
```suggestion
metadata specification. The reason for this is that some fields in the Hugging Face standard are too much
```

**Review by uittenbroekrobbert — APPROVED** [review]

File: docs/Projects/TAD/ADRs/0002-tad-reporting-standard.md (hunk 2 above)
Comment (ChristopherSpelt) [reply docs/Projects/TAD/ADRs/0002-tad-reporting-standard.md:15]: It does include fields for metrics, only less elaborate than our metric field. How about: 
```suggestion
This implementation does not capture sufficient details about metrics and does not include measurements from technical tests on bias en fairness or regulatory assessments.
```
?

File: docs/Projects/TAD/ADRs/0002-tad-reporting-standard.md (hunk 3 above)
Comment (anneschuth) [reply docs/Projects/TAD/ADRs/0002-tad-reporting-standard.md:25]: We may want to, in general, accept includes, and say something like that here too.

This is what we currently say [in our standard](https://minbzk.github.io/ai-validation/Projects/TAD/Reporting-Standard/#introduction):

> These model_cards and assessment_cards can be included verbatim into a system_card, or referenced with an !include statement, allowing for minimal cards to be compact in a single file. Extensive cards can be split up for readability and maintainability. Our standard allows for the !include to be used anywhere.

Comment (ChristopherSpelt) [reply docs/Projects/TAD/ADRs/0002-tad-reporting-standard.md:25]: In principle you can !include inside the system card any field you want. The standard prefers to do this for in any case the model card and assessment card. If we also want to prefer to include metrics and measurements via an include mechanism, we should specify this in the TAD standard. So for this ADR I would not make these suggestions.

File: docs/Projects/TAD/ADRs/0002-tad-reporting-standard.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+1. More finegrained information on performance metrics.
+2. Capturing additional measurements on fairness and bias.
+3. Capturing regulartory assessments.
+
+The extension is not strict, meaning that there the TAD Reporting Standard is not a valid Hugging Face
+metadata specification. The reason for this is that some fields in the Hugging Face standard are too much
+intertwined with the Hugging Face ecosystem and it would not be logical for us to couple our implementation
+this thighly to Hugging Face.
+
+The standard is written in yaml.
```
Comment (anneschuth) [inline docs/Projects/TAD/ADRs/0002-tad-reporting-standard.md:40]: I'm not sure we need to say this. The standard is more general, we just happen to have an implementation in yaml

**Review by anneschuth — DISMISSED** [review]
Nice, just a small comment

File: docs/Projects/TAD/ADRs/0002-tad-reporting-standard.md [hunk 6]
```diff
@@ -0,0 +1,51 @@
+# TAD-0002 TAD Reporting Standard
```
Comment (berrydenhartog) [inline docs/Projects/TAD/ADRs/0002-tad-reporting-standard.md:1]: I would add the TAD ADR in the TAD repo. Else we get a mess in all the ADRs and where they are.

Comment (anneschuth) [reply docs/Projects/TAD/ADRs/0002-tad-reporting-standard.md:1]: We don't really have a "TAD repo", or do we?

File: docs/Projects/TAD/ADRs/0002-tad-reporting-standard.md (hunk 2 above)
Comment (uittenbroekrobbert) [reply docs/Projects/TAD/ADRs/0002-tad-reporting-standard.md:15]: Yes, very good, I can not edit your suggestion but it should say: 'bias and fairness'.. going Duch there ;)

File: docs/Projects/TAD/ADRs/0002-tad-reporting-standard.md (hunk 3 above)
Comment (uittenbroekrobbert) [reply docs/Projects/TAD/ADRs/0002-tad-reporting-standard.md:25]: Ok, awesome, should have seen that ;)

File: docs/Projects/TAD/ADRs/0002-tad-reporting-standard.md (hunk 6 above)
Comment (berrydenhartog) [reply docs/Projects/TAD/ADRs/0002-tad-reporting-standard.md:1]: We do not but i would not add all ADRs in this repo. Would prefer to keep them as close as possible to the project.

Comment (ChristopherSpelt) [reply docs/Projects/TAD/ADRs/0002-tad-reporting-standard.md:1]: Decided to keep the current structure for now.

**Review by berrydenhartog — APPROVED** [review]
LGTM

**Pull request merged**